Assets/HoloToolkit/Utilities/Scripts/Editor/SceneSettingsWindow.cs

```diff
@@ -26,43 +31,31 @@ public enum SceneSetting
 
         protected override void ApplySettings()
         {
-            Camera mainCamera = CameraCache.Main;
-
-            // Ensure we have a camera
-            if (mainCamera == null)
+            if (Values[SceneSetting.AddMixedRealityCamera])
             {
-                Debug.LogWarning(@"Could not apply settings - no camera tagged with ""MainCamera""");
-                return;
+                Instantiate(AssetDatabase.LoadAssetAtPath<GameObject>(AssetDatabase.GUIDToAssetPath(CameraPrefabGUID)));
             }
 
-            // Apply individual settings
-            if (Values[SceneSetting.CameraToOrigin])
-            {
-                mainCamera.transform.position = Vector3.zero;
-            }
-
-            if (Values[SceneSetting.CameraClearBlack])
-            {
-                mainCamera.clearFlags = CameraClearFlags.SolidColor;
-                mainCamera.backgroundColor = Color.clear;
-            }
+            var mainCamera = CameraCache.Refresh(Camera.main);
 
-            if (Values[SceneSetting.NearClipPlane])
+            if (mainCamera == null)
             {
-                mainCamera.nearClipPlane = 0.85f;
+                Debug.LogWarning("Could not find a valid \"MainCamera\"!  Unable to update camera position.");
             }
-
-            if (Values[SceneSetting.FieldOfView])
+            else
             {
-                mainCamera.fieldOfView = 16.0f;
+                if (Values[SceneSetting.CameraToOrigin])
+                {
+                    mainCamera.transform.position = Vector3.zero;
+                }
             }
 
             Close();
         }
 
         protected override void LoadSettings()
         {
-            for (int i = (int)SceneSetting.CameraToOrigin; i <= (int)SceneSetting.FieldOfView; i++)
+            for (int i = (int)SceneSetting.CameraToOrigin; i <= (int)SceneSetting.CameraToOrigin; i++)
```

Comment:
This not starting from i=0 causes AddMixedRealityCamera to not appear in the menu

Reply:
Fixed in #1072

Reply:
I thought I had fixed this. Sorry about that. Must have slipped in my rebase retry.